sdk/digitaltwins/Azure.DigitalTwins.Core/src/Models/IDigitalTwinRelationship.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.DigitalTwins.Core
+{
+    /// <summary>
+    /// The service-defined fields of a digital twin relationship.
+    /// </summary>
+    public interface IDigitalTwinRelationship
+    {
+        /// <summary>
+        /// The unique Id of the relationship. This field is present on every relationship.
```

Comment:
You could add a note here saying what the json property name is that this property should correspond to ("$relationshipId" for this property)